mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -372,7 +372,6 @@
         // Spec GUI
         "MOM_WatchingReplay"    "REPLAY"
         "MOM_WatchingGhost"    "SPECTATING"
-        "MOM_ReplayPlayer"      "Player: %s1"
```

Comment:
Something to be careful of is making sure this token isn't used anywhere else. While this a spectator GUI specific string, some strings are shared among panels. Make sure to search other .res files for tokens if you ever remove them.

Reply:
I don't believe it's used anywhere else thankfully, but next time I will definitely be more careful. Thanks for the advice :)